src/common/services/DevService/DevService.ts

```diff
@@ -0,0 +1,9 @@
+export class DevService {
```

Comment:
Just some helper methods that are only needed for generating mock data. Could probably just merge this into `UtilService`.